src/models/orderDao.js

```diff
@@ -0,0 +1,80 @@
+const appDataSource = require("./dataSource");
+const queryRunner = appDataSource.createQueryRunner();
+
+const checkOrder = async (userId, bookId) => {
+  const getBookInfoByBookId = await appDataSource.query(
+    `
+  SELECT
+      b.title,
+      b.image_url,
+      b.author,
+      b.online_price
+      FROM books b   
+    WHERE b.id = ?
+      `,
+    [bookId]
+  );
+
+  const getPointsByUserId = await appDataSource.query(
+    `SELECT
+      u.point
+      FROM users u
+      WHERE u.id =?
+      `,
+    [userId]
+  );
+
+  return getBookInfoByBookId.concat(getPointsByUserId);
+};
+
+const createOrder = async (userId, bookId, online_price) => {
+  await queryRunner.connect();
+  await queryRunner.startTransaction();
+
+  try {
+    await queryRunner.query(
+      `
+        INSERT INTO bookshelves
+          (user_id, book_id, online_price)
+        VALUES
+        (?, ?, ?)
```

Comment:
가독성을 위해 다음과 같이 적을 수도 있을 것 같습니다.
```suggestion
        INSERT INTO
          bookshelves(
            user_id,
            book_id,
            online_price
          )
        VALUES (
          ?,
          ?,
          ?
        )
```